lib/rules/selector-disallowed-list/__tests__/index.mjs

```diff
@@ -248,6 +257,28 @@ testRule({
 			endLine: 1,
 			endColumn: 6,
 		},
+		{
+			code: '.color-0% {}',
```

Comment:
Is this valid CSS? If not, let's remove it or try to find a valid alternative.

Reply:
Great point, thanks - for some reason I thought `%` was allowed with an ident, my mistake!